lib/mergometer/report.rb

```diff
@@ -1,12 +1,16 @@
 require "progress_bar"
 require "mergometer/pull_request"
 
+reports_path = File.expand_path("./reports/**/*.rb", __FILE__)
+Dir[reports_path].each { |f| require f }
+
 module Mergometer
   class Report
     require "hirb"
 
-    def initialize(repo)
+    def initialize(repo, from = nil)
```

Comment:
I still feel like something like `(repo, **options={})` would be better here, and then allowing each sub report to define what kind of options it needs. Some reports might benefit from specifying `from: ...`, others might now?

Reply:
https://github.com/balvig/mergometer/pull/2#issuecomment-383803809

But I guess you are right, using option is more flexible!